docs/data/toolpad/getting-started/quickstart.md

```diff
@@ -46,72 +46,163 @@ Interactive walkthrough of the app building process:
 
    ```js
    dogQuery.data;
    ```
 
-   then click **Update binding**
+   save the binding and close the binding editor.
+
+   <div class="MuiToolpadDocs-ImageContainer" onClick="window.handleImageViewerOpen(event)">
+      <img src="/static/toolpad/docs/quickstart/step-8.png" alt="dogQuery.data" />
+      <span class="MuiToolpadDocs-EnlargeIcon">    
+      </span>
+   </div>
+
+   <p class="image-caption">The binding editor</p>
+
+1. If all went well, the Data Grid will now display the data from our query:
+
+   <div class="MuiToolpadDocs-ImageContainer" onClick="window.handleImageViewerOpen(event)">
+      <img src="/static/toolpad/docs/quickstart/step-9.png" alt="Connected data/>
+      <span class="MuiToolpadDocs-EnlargeIcon">    
+      </span>
+   </div>
+
+   <p class="image-caption">The Data Grid with bound data</p>
+
+### Make the app interactive
 
-   <img src="/static/toolpad/docs/quickstart/step-8.png?v=0" alt="dogQuery.data" width="1192" />
+1. Now, we can make this app interactive by displaying a random image of the selected breed. We'll create another query which reacts to the selection inside the Data Grid component. Deselect all components and click on **Add query** &rarr; **HTTP request**. Name it "imageQuery" and add a `breed` parameter in the **Parameters** section on the bottom:
 
-   If all went well, the DataGrid should now display the data from the HTTP request
+   <div class="MuiToolpadDocs-ImageContainer" onClick="window.handleImageViewerOpen(event)">
+      <img src="/static/toolpad/docs/quickstart/step-10.png" alt="Breed parameter" />
+      <span class="MuiToolpadDocs-EnlargeIcon">    
+      </span>
+   </div>
 
-   <img src="/static/toolpad/docs/quickstart/step-9.png" alt="Connected data" width="1512" />
+   <p class="image-caption">Editing imageQuery</p>
 
-1. Now let's make our app interactive by displaying a random image of the selected breed. We'll create a dynamic query which reacts to the selection inside Data Grid component. Click **Add query** and select "HTTP request" again. Name it "imageQuery" and add a parameter "breed"
+2. Click on the **Bind** button next to the breed parameter value, and add the following JavaScript expression in the binding editor:
 
-   <img src="/static/toolpad/docs/quickstart/step-10.png" alt="Breed parameter" width="707" />
+   ```jsx
+   dataGrid.selection?.[0] ?? 'akita';
+   ```
+
+   This will use the selected value from the Data Grid, and default to the "akita" breed when no row has been selected.
 
-   Bind the breed parameter value to `dataGrid.selection?.[0] ?? 'akita'`. This will use the selected value from `dataGrid` and defaults to the "akita" breed when no row has been selected.
+   <div class="MuiToolpadDocs-ImageContainer" onClick="window.handleImageViewerOpen(event)">
+      <img src="/static/toolpad/docs/quickstart/step-11.png" alt="Breed binding"  />
+      <span class="MuiToolpadDocs-EnlargeIcon">    
+      </span>
+   </div>
```

Comment:
Why override the MarkdownDocs and not render a custom component here? You can check MUI X codebase on how to do this, e.g. https://github.com/mui/mui-x/blob/master/docs/data/date-pickers/getting-started/getting-started.md?plain=1#L23